pkgs/desktops/kde-5/plasma/startkde/startkde.sh

```diff
@@ -144,6 +152,11 @@ if test -n "$kcminputrc_mouse_cursortheme" -o -n "$kcminputrc_mouse_cursorsize"
     fi
 fi
 
+# if we haven't defined a window decoration, use $THEME defined above
+write_default kwinrc org.kde.kdecoration2 library $THEME
```

Comment:
@peterhoeg IIUC, `write_default` just does what the `startupconfigkeys` file is for. Can you comment on why it's done differently here?